### PR TITLE
[EditText]Refactoring EditText

### PR DESCRIPTION
**Pull request by DevYuns**
## Description

Refactoring EditText. I'll make that the code has more readability and Scalability.

## Checklist

Before you create this PR confirms that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide](https://github.com/dooboolab/dooboo-ui/blob/master/CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] Run `yarn test` or `yarn test -u` if you need to update snapshot.
- [x] Run `yarn lint`
- [x] I am willing to follow-up on review comments in a timely manner.

### CODE REVIEW

**Review by hyochan — CHANGES_REQUESTED** [review]
image

I only see `default` where are other types??

**Comment by DevYuns** [comment]
I made default value only for code review. If It doesn't need any changes or improvement, I will make a more use case.

**Comment by hyochan** [comment]
@DevYuns  Please request review when you think you are ready

**Comment by DevYuns** [comment]
EditText

File: main/EditText/README.md [hunk 1]
```diff
@@ -56,104 +17,191 @@ yarn add @dooboo-ui/core
...
-          <StyledSignInButton
-            testID="btnEmail"
-            onPress={() => onSignIn()}
-            textStyle={fontStyle}
-            text="Login"
-          />
-          {/* Email SignUp text */}
-          <View
+          >
+            회원가입
```
Comment (hyochan) [inline main/EditText/README.md]: No Korean~

File: main/EditText/README.md [hunk 2]
```diff
@@ -56,104 +17,191 @@ yarn add @dooboo-ui/core
...
+          </Text>
+        </View>
+        <EditText
+          labelText={'이메일'}
+          value={email}
+          isErrored={isEmailErrored}
+          errorStyle={{ borderColor: '#E54E4E' }}
+          errorMessage={
+            <Text style={{ color: '#E54E4E', marginTop: 8 }}>
+              이메일이 유효하지 않습니다.
```
Comment (hyochan) [inline main/EditText/README.md]: No Korean~

**Review by hyochan — CHANGES_REQUESTED** [review]
Please remove Korean & re-request review!

**Review by hyochan — APPROVED** [review]

**Pull request merged**